### DISCUSSION TITLE
How to set which entities are enable in HA Discovery

### DISCUSSION THREAD

**Discussion by reefland** [Koenkk/zigbee2mqtt#10484] [General]
1.22.2-dev commit: 15f1df8

I have  ` last_seen: ISO_8601_local` set, and I see the timestamp included in the device states as such:
```
{
    "current": 0.73,
    "last_seen": "2022-01-01T10:53:28-05:00",
    "linkquality": 60,
    "power": 56.3,
    "state": "ON",
    "voltage": 121.1
}
```
in Home Assistant > MQTT Integration > Device, I can see the Zigbee device has disabled entities for `_last_seen` and even `_linkquality`.

I know I can manually enable these in HA, but where in Z2M can I define these attributes should be enabled by default so I don't have to do this manual work.  As new devices added, etc I don't want a manual step of remembering to enable these manually.

**Comment by Koenkk** [comment]
You can do this by adding the following to the z2m `configuration.yaml`

```yaml
device_options:
  homeassistant:
    last_seen:
      enabled_by_default: true
```

**Comment by reefland** [comment]
Thank You, I read the documentation on `device_options` several times before asking... the `enabled_by_default` doesn't seem to be documented.
